### DISCUSSION TITLE
42D26A52-CA95-D59C-06C6-16E653D1196B

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#87] [General]
# 42D26A52-CA95-D59C-06C6-16E653D1196B

This add-on adds over 77300 stars, 29 open clusters, and 38 globular clusters to the Large Magellanic Cloud. These stars are real, and the data is from SIMBAD. Specifically, the coordinates and names of the stars are real, and the spectral types (if known) have been included.

https://celestia.mobi/addon?item=42D26A52-CA95-D59C-06C6-16E653D1196B&lang=en

**Comment by julianvinicio** [comment]
Jv is smart
Jv is amazing
